### PR TITLE
Lmm/testpy3

### PR DESCRIPTION
**Pull request by morenol**

### CODE REVIEW

File: .travis/docker-compose-travis.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: "2"
 
 services:
   analytics_api:
-    image: edxops/analytics_api:latest
+    image: ednxops/analytics_api:py3
```
Comment (marcotuts) [inline .travis/docker-compose-travis.yml:5]: is the n in ednxops intentional?

Comment (marcotuts) [reply .travis/docker-compose-travis.yml:5]: ah that's for your repo :) i see.

Comment (felipemontoya) [reply .travis/docker-compose-travis.yml:5]: yes, that is our hubspot account. Sometimes we use `ednx_` as a prefix for edunext owned repos and in this case it turned out to be confusingly similar to edx's edxops account.

Comment (felipemontoya) [reply .travis/docker-compose-travis.yml:5]: However, this PR is never to be merged. It is only here to make it easy for us to find the results of running travis

**Comment by tusbar** [comment]
# [Codecov](https://codecov.io/gh/eduNEXT/edx-analytics-data-api/pull/1?src=pr&el=h1) Report
> Merging [#1](https://codecov.io/gh/eduNEXT/edx-analytics-data-api/pull/1?src=pr&el=desc) into [master](https://codecov.io/gh/eduNEXT/edx-analytics-data-api/commit/e2ec468716346ede100c8a590f68d32ea9ec75f8&el=desc) will **decrease** coverage by `0.02%`.
> The diff coverage is `92.63%`.

[Impacted file tree graph](https://codecov.io/gh/eduNEXT/edx-analytics-data-api/pull/1?src=pr&el=tree)

```diff
@@            Coverage Diff             @@
##           master       #1      +/-   ##
==========================================
- Coverage   95.39%   95.36%   -0.03%     
==========================================
  Files          53       53              
  Lines        3384     3384              
  Branches      256      294      +38     
==========================================
- Hits         3228     3227       -1     
  Misses        130      130              
- Partials       26       27       +1     
```


| [Impacted Files](https://codecov.io/gh/eduNEXT/edx-analytics-data-api/pull/1?src=pr&el=tree) | Coverage Δ | |
|---|---|---|
| [analytics\_data\_api/v0/views/\_\_init\_\_.py](https://codecov.io/gh/eduNEXT/edx-analytics-data-api/pull/1/diff?src=pr&el=tree#diff-YW5hbHl0aWNzX2RhdGFfYXBpL3YwL3ZpZXdzL19faW5pdF9fLnB5) | `97.14% <0.00%> (-0.96%)` | :arrow_down: |

------

[Continue to review full report at Codecov](https://codecov.io/gh/eduNEXT/edx-analytics-data-api/pull/1?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/eduNEXT/edx-analytics-data-api/pull/1?src=pr&el=footer). Last update [e2ec468...ce21fdd](https://codecov.io/gh/eduNEXT/edx-analytics-data-api/pull/1?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by morenol** [comment]
Python 3 tests were merged in upstream.

**Pull request closed without merging**